installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,215 @@
+
+* Save the file as `$XDG_RUNTIME_DIR/containers/auth.json`.
+
+
+[id="using-oc-mirror"]
+== Using oc-mirror
+
+The instructions in this section will show how to prepare the environment and use oc-mirror.
+
+.Environment Preparation
+
+* To set up the credentials for pulling/pushing container images, download the pull secret and place it in the correct path.
+
+** The default podman credentials location `$XDG_RUNTIME_DIR/containers/auth` is used for authenticating to the registries.
+** The docker location `~/.docker/config.json` for credentials is also supported.
+
+[Optional Step] 
+* Use the following command to run a local registry that will be used as a destination:
+
+[source,terminal]
+----
+$ podman run -dt -p 6000:5000 -e REGISTRY_STORAGE_DELETE_ENABLED=true --name local-registry docker.io/library/registry:2
+----
+
+[id="building-image-set-configuration"]
+== Building the image set configuration
+
+The Image Set Configuration is the input file used by oc-mirror to discover which images are necessary to be mirrored.
+
+Below is an example which included an Openshift Release (version 4.13.10), an operator catalog with only 3 operators and some additional images (ubi8 and ubi9):
+
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v2alpha1
+mirror:
+  platform:
+    channels:
+    - name: stable-4.13
+      minVersion: 4.13.10
+      maxVersion: 4.13.10
+    graph: true
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+      packages:
+       - name: aws-load-balancer-operator
+       - name: 3scale-operator
+       - name: node-observability-operator
+  additionalImages: 
+   - name: registry.redhat.io/ubi8/ubi:latest
+   - name: registry.redhat.io/ubi9/ubi@sha256:20f695d2a91352d4eaa25107535126727b5945bff38ed36a3e59590f495046f0
+----
+
+More Image Set Configuration example can be found in the additional resources.
+
+[id="oc-mirror-workflows"]
+== oc-mirror workflows
+
+This section will explain all the workflows supported by oc-mirror currently.
+
+.Mirror to Disk
+
+Pulls the container images from the source specified in the image set configuration and pack them into a tar archive on disk (local directory).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml <file path> --v2
+----
+
+.Disk to Mirror
+
+Copy the containers images from the tar archive to a container registry (--from flag is required on this workflow).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml --from file:///oc-mirror/mirror1 docker://localhost:6000 --v2
+----
+
+.Mirror To Mirror
+
+Copy the container images from the source specified in the image set configuration to the destination (container registry).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml --workspace ffile:///oc-mirror/mirror1 docker://localhost:6000 --v2
```

Comment:
Typo in the command file. s/ffile/file.

@sherine-k do we need to add --dest-tls-verify=false as well or do you think user or customer would know based on the error they receive ?

Reply:
We should always encourage https over http , so maybe wording to say use --help if using an insecure registry

Reply:
Done